deployment/docker/docker-compose-linux.yml

```diff
@@ -3,7 +3,7 @@ version: '3.7'
 
 services:
   forms-flow-bpm-db:
-    image: postgres:latest
+    image: postgres:14
```

Comment:
Also I see latest camunda version still does not support 14 version of postgres.
https://docs.camunda.org/manual/latest/introduction/supported-environments/

![image](https://user-images.githubusercontent.com/62901416/164297085-bcb13382-45cc-4613-974a-648060e90aef.png)
Please verify

Reply:
Great point, but since we working on higher versions already, should we downgrade it to 13?

Reply:
I think we need to change it accordingly as per the recommendation of Camunda, so it should be downgraded. Also I see postgres 14 is still in its inital phase, nothing major in the release notes different from the 13.6 version. So its better to downgrade it to 13 overall. I think even BC GOV is still in postgres 12.